client/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "superdesk-client",
   "license": "GPL-3.0",
   "dependencies": {
-    "superdesk-core": "superdesk/superdesk-client-core#b3988c8c0b9aeb08896d03ca489588529a41aa78",
+    "superdesk-core": "vladnicoara/superdesk-client-core#48346a37115f2e38e4325ed76785136cdbccd862",
```

Comment:
To validate if the PR from core client pass all e2e tests. This PR is just for validation it will not be merged.